recipes/cookies/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "cookies" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 6f4c53aba3ed03e4e7b50812c2c2579a
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  commands:
+    - 'py.test {{ SRC_DIR }}{{ os.sep }}test*'
+
+about:
+  home: https://gitlab.com/sashahart/cookies
+  license: MIT
+  summary: >
+    Friendlier RFC 6265-compliant cookie parser/renderer.  It can be used as
+    a replacement of Python’s Cookie.py (aka http.cookies).
```

Comment:
The linter might be choking on that unicode `'`. (I thought that was fixed :confused:)